pyproject.toml

```diff
@@ -25,37 +25,29 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<3.12"
+python = ">=3.8,<3.12"
 numpy = [
-    { version = ">=1.19", python = ">=3.7,<3.11" },
+    { version = ">=1.19", python = ">=3.8,<3.11" },
     { version = ">=1.23", python = ">=3.11" },
 ]
 pandas = [
-    { version = ">=1.1,<1.6", python = ">=3.7,<3.11" },
+    { version = ">=1.1,<1.6", python = ">=3.8,<3.11" },
```

Comment:
Non-blocking question: can this be simplified? something like `pandas = "<1.6"`?